### PR TITLE
Treat int columns in the right way

### PR DESCRIPTION
**Pull request by foxtrot-dfm1**
Deserializer can’t t convert the int fields, so I used the plone.restapi convert technique to solve the problem

### CODE REVIEW

**Review by petschki — CHANGES_REQUESTED** [review]
Could you please rebase this onto master -> tests will be green then.

Also a test which proves your fix would be very cool.

File: CHANGES.rst [hunk 1]
```diff
@@ -4,7 +4,8 @@ Changelog
 3.0.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Treat the int columnts in the right way by deserializer.
```
Comment (gforcada) [inline CHANGES.rst:7]:
```suggestion
- Treat the int columns in the right way by deserializer.
```

As I was asked a review, at least I can spot typos 😄

**Review by gforcada — COMMENTED** [review]
Only looking at the patch I'm rather at a loss to say if it's good or not, as there is no test to check that it fixes something.

Would it be possible to get one?

There are some basic tests on https://github.com/collective/collective.z3cform.datagridfield/blob/master/src/collective/z3cform/datagridfield/browser.rst

File: src/collective/z3cform/datagridfield/row.py [hunk 2]
```diff
@@ -58,6 +59,10 @@ def _validate(self, value):
                 bound = field_type.bind(self.context)
                 bound.validate(value[field_name])
             else:
+                if isinstance(value[field_name], str):
+                    value[field_name] = IFromUnicode(field).fromUnicode(
+                        value[field_name]
```
Comment (foxtrot-dfm1) [inline src/collective/z3cform/datagridfield/row.py:64]: must be field_type

**Pull request closed without merging**